### PR TITLE
Add a New Button

### LINKED ISSUE #311: Issue: Add a New Button
**This is a(n):**

- [ ] Update to an existing Animated Button
- [ ] Any Error
- [x] Proposal to the Repository
- [ ] Documentation / README.md changes

**Details:**

I want to contribute this button as part of GSSoC'23.

### PR DESCRIPTION
**Pull request by Sriparno08**
Added a new animated button, created using HTML & CSS.

Part of: GSSoC'23

Fixes #311

### CODE REVIEW

File: src/Data.js [hunk 1]
```diff
@@ -119,6 +119,7 @@ export const Data = [
   "Spyware007_4",
   "Spyware007_5",
   "Spyware007_6",
+  "Sriparno08";
```
Comment (Spyware007) [inline src/Data.js:122]: A comma should be present here!

**Comment by Spyware007** [comment]
Please Pull the branch first as other PRs getting merged too...

**Comment by Sriparno08** [comment]
Hey, fixed it. Have a look!

**Comment by Spyware007** [comment]
Change your styles.css to style.css and it will be fixed!

**Comment by Sriparno08** [comment]
Done!

**Comment by Sriparno08** [comment]
Fixed!

**Comment by Spyware007** [comment]
@Sriparno08 Amazing work.
Thank you so much for your contribution.🌟

Make sure to ⭐ and follow me.
 I would love to connect with you all ❤️
 https://www.linkedin.com/in/om-gawande/

**Pull request merged**